components/dashboard/public/styles.css

```diff
@@ -826,7 +826,6 @@ footer .logo-icon {
 
 
 .access-control__card-container {
-    width: 31%;
```

Comment:
fixing a bogus line wrap occurring with longer hostnames of self-hosted git providers.